http/src/main/resources/reference.conf

```diff
@@ -100,6 +100,7 @@ monitor {
   pollPeriod = 15 seconds
   maxRetries = -1  # means retry forever
   recreateCluster = true
+  creationTimeLimit = 20 minutes
```

Comment:
generous window given to not flag hanging clusters prematurely. Custom images may take longer, so this may need to be bumped up in the future, but this should give a good tolerance

Reply:
We may want to bump this higher by default - maybe 1 hour? A lot of users specify user scripts which can take a long time to execute.